frames/src/main/java/com/checkout/frames/cvvcomponent/CvvInputComponent.kt

```diff
@@ -0,0 +1,11 @@
+package com.checkout.frames.cvvcomponent
+
+import androidx.compose.runtime.Composable
+import com.checkout.frames.cvvcomponent.models.CVVComponentConfig
+
+@Suppress("UnusedPrivateMember")
+@Composable
+internal fun CvvInputComponent(config: CVVComponentConfig) {
```

Comment:
Let's align `cvv` to all upper case?

```suggestion
internal fun CVVInputComponent(config: CVVComponentConfig) {
```